src/components/common/Ellipsis.tsx

```diff
@@ -0,0 +1,11 @@
+import { styled } from '@mui/system';
+import { Link } from 'react-router-dom';
+import { CustomTheme } from '@mui/material/styles';
+
+export const EllipsisLink = styled(Link)(({ theme }) => ({
+  ...(theme as CustomTheme).mixins.ellipsis,
+}));
```

Comment:
Oh nice! This allows the format to match between sx props and styled components.